corehq/apps/dump_reload/couch/dump.py

```diff
@@ -23,7 +23,6 @@
     DocTypeIDProvider(['Application']),
     DocTypeIDProvider(['CommtrackConfig']),
     DocTypeIDProvider(['DefaultConsumption']),
-    ViewIDProvider('CommCareMultimedia', 'hqmedia/by_domain', DomainKeyGenerator()),
```

Comment:
@orangejenny do you know whether there was a plan to replace this?  I see the view was [later removed too](https://github.com/dimagi/commcare-hq/pull/26007/files).  Without this view, I don't see a way to get multimedia accessible to a specific domain.  Looks like `linked_domain` uses the multimedia associated with individual apps, though that seems tricky, as it only accounts for current app versions.

In order for this command to keep working, it looks like we'll need to re-add the couch view, unless you've got a trick up your sleeve.

Reply:
Guessing I removed this because I didn't clearly understand what dump is for. Looking over this code again, yes, I think you're right it needs to be restored until multimedia is migrated to SQL.

I think it will be quick to PR the restore, but if it isn't (or you're just not in the mood) I'm happy to take it.

Reply:
Looks like Simon already added it back in:
https://github.com/dimagi/commcare-hq/pull/27283
https://github.com/dimagi/commcare-hq/pull/27285
:star: